website/www/site/content/en/get-started/downloads.md

```diff
@@ -87,6 +87,13 @@ versions denoted `0.x.y`.
 
 ## Releases
 
+### 2.21.0 (2020-05-27)
+Official [source code download](http://www.apache.org/dyn/closer.cgi/beam/2.21.0/apache-beam-2.21.0-source-release.zip).
+[SHA-512](https://downloads.apache.org/beam/2.21.0/apache-beam-2.21.0-source-release.zip.sha512).
+[signature](https://downloads.apache.org/beam/2.21.0/apache-beam-2.21.0-source-release.zip.asc).
+
+[Release notes](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12319527&version=12347143).
+
 ### 2.20.0 (2020-04-15)
 Official [source code download](http://www.apache.org/dyn/closer.cgi/beam/2.20.0/apache-beam-2.20.0-source-release.zip).
```

Comment:
You will need to update 2.20.0 links to `archive`. See the examples of 2.19.0

Reply:
Done. (I'll wait to merge this since files haven't been moved yet.)